config/jobs/kubernetes/sig-node/dra-ci.yaml

```diff
@@ -127,7 +119,6 @@ periodics:
       repo: kubernetes
       base_ref: master
       path_alias: k8s.io/kubernetes
-      workdir: true
```

Comment:
I am not sure whether it is safe to remove this. Perhaps keep it, to be explicit and to avoid a potential regression in the periodic job after merging?

Reply:
I'm not sure either, just wanted to try. Returned back.